eland/tests/dataframe/test_head_tail_pytest.py

```diff
@@ -95,3 +96,9 @@ def test_doc_test_tail(self):
         df = df[["timestamp", "OriginAirportID", "DestAirportID", "FlightDelayMin"]]
         df = df.tail()
         print(df)
+
+    def test_doc_test_tail_empty(self):
+        df = self.ed_flights()
+        df = df[df.OriginAirportID == "NADA"]
+        df = df.tail()
+        assert df.shape[0] == 0
```

Comment:
Can we add a case for n=1 as well?

Reply:
By n=1 you mean `df.tail(1)` ?

Reply:
Oh sorry I meant a case where the dataframe contains 1 row so calling `df.tail().shape[0] == 1`

Reply:
Done!